src/Ledger/Ledger/Properties.agda

```diff
@@ -99,39 +99,69 @@ Computational-LEDGERS = it
 
 instance
   HasCoin-LState : HasCoin LState
-  HasCoin-LState .getCoin s = getCoin (LState.utxoSt s)
+  HasCoin-LState .getCoin s = getCoin (LState.utxoSt s) + getCoin (LState.certState s)
 
 -- ** Proof that LEDGER preserves values.
 
 FreshTx : Tx → LState → Type
 FreshTx tx ls = txid ∉ mapˢ proj₁ (dom (ls .utxoSt .utxo))
   where open Tx tx; open TxBody body; open UTxOState; open LState
 
-module _ (tx : Tx) (let open Tx tx; open TxBody body) where
+module _
+  (tx : Tx) (let open Tx tx; open TxBody body)
+  ( indexedSumᵛ'-∪  :  {A : Type} ⦃ _ : DecEq A ⦄ (m m' : A ⇀ Coin)
+                       → disjoint (dom m) (dom m') → getCoin (m ∪ˡ m') ≡ getCoin m + getCoin m' )
+  ( sumConstZero    :  {A : Type} ⦃ _ : DecEq A ⦄ {X : ℙ A} → getCoin (constMap X 0) ≡ 0 )
+  ( res-decomp      :  {A : Type} ⦃ _ : DecEq A ⦄ (m m' : A ⇀ Coin)
+                       → (m ∪ˡ m')ˢ ≡ᵉ (m ∪ˡ (m' ∣ dom (m ˢ) ᶜ))ˢ )
+  ( getCoin-cong    :  {A : Type} ⦃ _ : DecEq A ⦄ (s : A ⇀ Coin) (s' : ℙ (A × Coin)) → s ˢ ≡ᵉ s'
+                       → indexedSum' proj₂ (s ˢ) ≡ indexedSum' proj₂ s' )
+  ( ≡ᵉ-getCoinˢ     :  {A A' : Type} ⦃ _ : DecEq A ⦄ ⦃ _ : DecEq A' ⦄ (s : ℙ (A × Coin)) {f : A → A'}
+                       → InjectiveOn (dom s) f → getCoin (mapˢ (map₁ f) s) ≡ getCoin s )
+  ( constNetworkId  :  (wdls : RwdAddr ⇀ Coin) → ∀[ a ∈ dom (wdls ˢ) ] RwdAddr.net a ≡ NetworkId )
```

Comment:
This follows from the following line in `UTXO-inductive`:
```
∙ ∀[ a ∈ dom txwdrls ]          a .RwdAddr.net  ≡ networkId
```

Reply:
The problem here is that we have multiple `NetworkId`s preventing a unification in one of the proofs.  I'll remove one of the `NetworkId`s from `globalConstants`.